File: JS/2469.convert-the-temperature.js
/*
 * @lc app=leetcode id=2469 lang=javascript
 *
 * [2469] Convert the Temperature
 */

// @lc code=start
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function (celsius) {
  return [(celsius + 273.15).toFixed(5), (celsius * 1.8 + 32.0).toFixed(5)];
};
// @lc code=end

// console.log(convertTemperature(122.11));
// console.log(convertTemperature(36.5));

File: GO/2469.convert-the-temperature.go
/*
 * @lc app=leetcode id=2469 lang=golang
 *
 * [2469] Convert the Temperature
 */
package main

// @lc code=start
func convertTemperature(celsius float64) []float64 {
	return []float64{celsius + 273.15, (celsius * 1.80) + 32.00}
}

// @lc code=end
